models/model.js

```diff
@@ -42,3 +42,24 @@ exports.selectUsers = () => {
             return users;
         })
 }
+
+exports.updateVotebyArticleId = (voteChange, article_id, voteObjKey) => {
+
```

Comment:
it seems like you're not actually using `voteObjKey` so it isn't needed

Reply:
I will refactor it once I've done the other tasks. Thanks Heather.